Foundation/NSSwiftRuntime.swift

```diff
@@ -25,6 +25,9 @@ import CoreFoundation
 #endif
 
 public typealias ObjCBool = Bool
```

Comment:
Can/should this be a struct as documented? https://developer.apple.com/documentation/objectivec/objcbool

Reply:
@alblue This is part of making the unit tests run on native Darwin Foundation.
@ianpartridge I suspect yes, it may require a full implementation to match Darwin's foundation, I will investigate it.